examples/01_1DFFT/01_1DFFT.cpp

```diff
@@ -24,23 +24,29 @@ int main(int argc, char* argv[]) {
 
     Kokkos::Random_XorShift64_Pool<> random_pool(12345);
     Kokkos::fill_random(xc2c, random_pool, I);
+    Kokkos::fence();
 
     KokkosFFT::fft(execution_space(), xc2c, xc2c_hat);
     KokkosFFT::ifft(execution_space(), xc2c_hat, xc2c_inv);
+    Kokkos::fence();
```

Comment:
execution space fences are sufficient
```suggestion
    exec.fence();
```